setpass.py

```diff
@@ -0,0 +1,45 @@
+#   Copyright 2016 Massachusetts Open Cloud
+#
+#   Licensed under the Apache License, Version 2.0 (the "License"); you may
+#   not use this file except in compliance with the License. You may obtain
+#   a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#   WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#   License for the specific language governing permissions and limitations
+#   under the License.
+import string
+import random
+
+
+def random_password(size):
+    """Generate a random password of length 'size'"""
+    chars = string.ascii_letters + string.digits + string.punctuation[2:6]
```

Comment:
Just because this code is getting dug up again, let me re-iterate an old comment: https://github.com/CCI-MOC/moc/pull/57/files/4b48ea41400ab2b32f306782e42425a762a583bf#diff-4300e27fa37d58fbf3ed1d981980c150R30  

Especially since we have a docstring now, we really should be clearer about the meaning of `string.punctuation[2:6]`